docs/source/markdown/podmansh.1.md

```diff
@@ -42,7 +44,7 @@ Image=registry.fedoraproject.org/fedora
 ContainerName=podmansh
 RemapUsers=keep-id
 RunInit=yes
-DropCapabilities=all
```

Comment:
Just double checkins, IDK, but is this change correct?

Reply:
Yes this is the major reason for the change.  I am working on testing out the three scenarios and beginning to write a blog/howto for using podmanh.